packages/rainbowkit/src/components/MenuButton/MenuButton.css.ts

```diff
@@ -1,25 +1,6 @@
 import { style } from '@vanilla-extract/css';
-import { sprinkles } from '../../css/sprinkles.css';
 
-export const MenuButtonClassName = style([
```

Comment:
As part of an earlier solution attempt, I moved these styles to be Box props rather than classes so I was able to inspect the transform being applied. I decided this was worthwhile clean up so I left it as part of this PR.